src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -8760,13 +8784,17 @@ def applymap(
             return self._apply_snowpark_python_function_to_columns(func, kwargs)
 
         # Check if the function is a known numpy function that can be translated to Snowflake function.
-        sf_func = NUMPY_FUNCTION_TO_SNOWFLAKE_FUNCTION.get(func)
+        sf_func = NUMPY_UNIVERSAL_FUNCTION_TO_SNOWFLAKE_FUNCTION.get(func)
         if sf_func is not None:
             # TODO SNOW-1739034: remove pragma no cover when apply tests are enabled in CI
             return self._apply_snowpark_python_function_to_columns(
                 sf_func, kwargs
             )  # pragma: no cover
 
+        if func in (np.sum, np.min, np.max):
+            # Aggregate functions applied element-wise to columns are no-op.
+            return self  # pragma: no cover
```

Comment:
why `# pragma: no cover`?

Reply:
Right now apply tests don't run due to CI. so code coverage check fails. This is conversation thread https://snowflake.slack.com/archives/C04HF38JFAQ/p1734113965907879